docs/MO_DG/prepare_model/Getting_performance_numbers.md

```diff
@@ -1,28 +1,25 @@
 # Getting Performance Numbers {#openvino_docs_MO_DG_Getting_Performance_Numbers}
 
+When evaluating performance of a model with the OpenVINO Runtime, it is required to measure the proper set of operations. To do so, consider the tips below: 
```

Comment:
This guide introduces things to notice and how to use the Benchmark.app to get performance numbers. In the last section, it also explains how the performance numbers are reflected through internal inference performance counters and execution graphs.